paddle/fluid/operators/softmax_with_cross_entropy_op.cu

```diff
@@ -58,8 +63,17 @@ __global__ void SoftCrossEntropyGradientKernel(T* logit_grad,
 
 }  // namespace
 
+static __device__ __forceinline__ platform::float16 real_exp(
```

Comment:
same here.

Reply:
Just renamed `exp` and removed `real_log`